todo_app/data/trello_items.py

```diff
@@ -0,0 +1,83 @@
+import requests, json, os
+from todo_app.data.item_class import Item
+
+board_id = os.getenv('BOARD_ID')
+api_key = os.getenv('TRELLO_API_KEY')
+token = os.getenv('TRELLO_API_TOKEN')
+to_do_list = os.getenv('TO_DO_LIST_ID')
+doing_list = os.getenv('DOING_LIST_ID')
+done_list = os.getenv('DONE_LIST_ID')
+
+
+def trello_get_items():
+    url = 'https://api.trello.com/1/boards/{board_id}/lists'.format(board_id=board_id)
+    headers = {
+        "Accept": "*/*"
+    }
+
+    query = {
+        'key': api_key,
+        'token': token,
+        'cards': 'open'
+    }
+    response = requests.request(
+        "GET",
+        url,
+        headers=headers,
+        params=query
+    )
+    lists = json.loads(response.text)
+    items = []
+    for listOfItems in lists:
+        status = listOfItems["name"]
+        tasks = listOfItems["cards"]
+        for task in tasks:
+            item = Item.from_trello_card(card={'id': task['id'], 'status': status, 'name': task['name']})

```

Comment:
It's slightly odd here how we unpack the task and then create a very similar card item. E.g. we could achieve the same by fudging the task:
```python
        for task in tasks:
            task['status'] = status
            item = Item.from_trello_card(card=task)
```
Or just allow the list name to be passed through as a separate argument